src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -154,6 +154,7 @@ public async Task InitAsync()
             bool shouldRetry;
 
             var random = new Random();
+            var startRegisterConnection = DateTime.UtcNow.Ticks;
```

Comment:
Wouldn't be more elegant / easier to use `StopWatch`?

Reply:
yes